migrations/3_batch_auction.js

```diff
@@ -1,16 +1,13 @@
 /*eslint no-undef: "off"*/
 
 const BiMap = artifacts.require("IdToAddressBiMap.sol")
-const IterableAppendOnlySet = artifacts.require("IterableAppendOnlySet.sol")
 const SnappBaseCore = artifacts.require("SnappBaseCore.sol")
 const SnappAuction = artifacts.require("SnappAuction.sol")
 
 module.exports = async function (deployer) {
   await deployer.deploy(BiMap)
-  await deployer.deploy(IterableAppendOnlySet)
 
   await deployer.link(BiMap, SnappBaseCore)
-  await deployer.link(IterableAppendOnlySet, SnappBaseCore)
```

Comment:
I am not sure I understand what is intended by removing this linkage. OH!! Maybe we need to change the `` address in the `common.env` as a result of this new linkage...